src/SimplCommerce.WebHost/appsettings.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=SimplCommerce;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
We shouldn't commit the change in ConnectionStrings, it's specific to developers. We recommend using secret manager http://docs.simplcommerce.com/en/latest/development-tips/

Reply:
My bad, don't know how I missed that. I will update my code now